### PR TITLE
Change bot.say/me functions out for bot.safe_say/safe_me

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable

### CODE REVIEW

File: pajbot/modules/basic/stream_update.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def generic_update(self, bot: Bot, source, message: str, field: str, extra_args:
             return
 
         log_msg = f'{source} updated the {field} to "{message}"'
-        bot.say(log_msg)
+        bot.safe_say(log_msg)
```
Comment (pajlada) [inline pajbot/modules/basic/stream_update.py:73]: No need to run safe_say on moderator-only commands.
The risks we mitigate normally with the change from `say` to `safe_say` is if users have banphrased usernames or are using the bot to say banphrased messages. I believe moderators can be trusted to use moderator commands not to abuse banphrases (they could just remove banphrases), so the potential confusion if a game name is banphrased and no response is given is not worth it.

File: pajbot/modules/basic/stream_update.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def update_game(self, bot: Bot, source, message, **rest) -> Any:
         # Resolve game name to game ID
         game: Optional[TwitchGame] = self.bot.twitch_helix_api.get_game_by_game_name(message)
         if not game:
-            bot.say(f"Unable to find a game with the name '{message}'")
+            bot.safe_say(f"Unable to find a game with the name '{message}'")
```
Comment (pajlada) [inline pajbot/modules/basic/stream_update.py:84]: No need to run safe_say on moderator-only commands.

File: pajbot/modules/chat_alerts/livealert.py [hunk 3]
```diff
@@ -44,9 +44,9 @@ def on_stream_start(self, **rest):
         streamer = self.bot.streamer_display
         game = self.bot.stream_manager.game
         title = self.bot.stream_manager.title
-        self.bot.say(live_chat_message.format(streamer=streamer, game=game, title=title))
+        self.bot.safe_say(live_chat_message.format(streamer=streamer, game=game, title=title))
         if self.settings["extra_message"] != "":
-            self.bot.say(self.settings["extra_message"].format(streamer=streamer))
+            self.bot.safe_say(self.settings["extra_message"].format(streamer=streamer))
```
Comment (pajlada) [inline pajbot/modules/chat_alerts/livealert.py:49]: No need to run safe_say on this as it's triggered by a broadcaster and moderators actions.
Moderator sets the message, streamer sets the game/title.

File: pajbot/modules/chat_alerts/livealert.py [hunk 4]
```diff
@@ -44,9 +44,9 @@ def on_stream_start(self, **rest):
         streamer = self.bot.streamer_display
         game = self.bot.stream_manager.game
         title = self.bot.stream_manager.title
-        self.bot.say(live_chat_message.format(streamer=streamer, game=game, title=title))
+        self.bot.safe_say(live_chat_message.format(streamer=streamer, game=game, title=title))
```
Comment (pajlada) [inline pajbot/modules/chat_alerts/livealert.py:47]: No need to run safe_say on this as it's triggered by a broadcaster and moderators actions.
Moderator sets the message, streamer sets the game/title.

**Review by pajlada — CHANGES_REQUESTED** [review]

**Comment by RAnders00** [comment]
Instead of doing the shotgun replace-all-in-path approach, I would like you to please go through each of the replacements manually. What you're basically doing is replacing every single one and then making us review every single one. If you want to change say to safe_say and me to safe_me then the work of checking every single one is on you.

**Comment by alazymeme** [comment]
> Instead of doing the shotgun replace-all-in-path approach, I would like you to please go through each of the replacements manually. What you're basically doing is replacing every single one and then making us review every single one. If you want to change say to safe_say and me to safe_me then the work of checking every single one is on you.

Actually if you take a look at the commit history, you can see me doing the "shotgun replace all approach" then fixing the ones that werent necessary. It just so happened that I missed a few.

**Pull request closed without merging**